x-pack/plugins/cloud_security_posture/server/routes/benchmarks/benchmarks.ts

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { uniq, map } from 'lodash';
+import type { IRouter, SavedObjectsClientContract } from 'src/core/server';
+import { schema as rt, TypeOf } from '@kbn/config-schema';
+import { transformError } from '@kbn/securitysolution-es-utils';
+import { string } from 'io-ts';
+import {
+  PackagePolicyServiceInterface,
+  AgentPolicyServiceInterface,
+  AgentService,
+} from '../../../../fleet/server';
+import { GetAgentPoliciesResponseItem, PackagePolicy, AgentPolicy } from '../../../../fleet/common';
+import { BENCHMARKS_ROUTE_PATH, CIS_VANILLA_PACKAGE_NAME } from '../../../common/constants';
+import { CspAppContext } from '../../plugin';
+
+export const isNonNullable = <T extends unknown>(v: T): v is NonNullable<T> =>
+  v !== null && v !== undefined;
+
+type BenchmarksQuerySchema = TypeOf<typeof benchmarksInputSchema>;
+
+export interface Benchmark {
+  package_policy: Pick<
+    PackagePolicy,
+    | 'id'
+    | 'name'
+    | 'policy_id'
+    | 'namespace'
+    | 'package'
+    | 'updated_at'
+    | 'updated_by'
+    | 'created_at'
+    | 'created_by'
+  >;
+  agent_policy: Pick<GetAgentPoliciesResponseItem, 'id' | 'name' | 'agents'>;
+}
+
+export const DEFAULT_BENCHMARKS_PER_PAGE = 1000;
+
+export const getPackagePolicies = async (
+  soClient: SavedObjectsClientContract,
+  packagePolicyService: PackagePolicyServiceInterface | undefined,
+  packageName: string,
+  queryParams: BenchmarksQuerySchema
+): Promise<PackagePolicy[]> => {
+  if (!packagePolicyService) {
+    throw new Error('packagePolicyService is undefined');
+  }
+
+  const { items: packagePolicies } = (await packagePolicyService?.list(soClient, {
+    kuery: `ingest-package-policies.package.name:${packageName}`,
```

Comment:
use `PACKAGE_POLICY_SAVED_OBJECT_TYPE` instead of `ingest-package-policies`
```suggestion
    kuery: `${PACKAGE_POLICY_SAVED_OBJECT_TYPE}.package.name:${packageName}`,
```